src/components/loader/readme.md

```diff
@@ -8,40 +8,39 @@ The `calcite-loader` component can act as a determinate or indeterminate loading
 
 ### Basic
 
-You can use the `active` attribute to toggle visibility:
+You can use the global attribute hidden to toggle visibility:
```

Comment:
Do we need this doc? This could apply to any component. `hidden` is a global attribute that would hide any component. Just seems odd to have that doc here.

Reply:
Agreed.